config.json

```diff
@@ -108,11 +108,25 @@
         "strings"
       ]
     },
+    {
+      "slug": "acronym",
+      "uuid": "3f8d18d4-4f31-405b-8aad-80f768b11db3",
+      "core": false,
+      "unlocked_by": "hamming",
+      "difficulty": 2,
+      "topics": [
+        "control_flow_conditionals",
+        "control_flow_loops",
+        "equality",
+        "regular_expressions",
+        "strings"
+      ]
+    },
     {
       "slug": "phone-number",
       "uuid": "8649b2b4-02f6-4947-92b9-d1cc0d7c3594",
       "core": false,
-      "unlocked_by": "word-count",
```

Comment:
Does this mean a new exercise needs to be unlocked by word-count other than phone-number?

Reply:
Core exercises **cannot** be unlocked by another, so the order of core exercises matters as far as how the user will progressed through them,

Non-core exercises, I believe, **have to** be unlocked by another exercise. The `configlet` tool will check for anything that doesn't meet these conditions.